### PR TITLE
CFY-3717 copy roles mapping to manager on bootstrap

### PR DESCRIPTION
**Pull request by adamlavie**

### CODE REVIEW

File: components/restservice/scripts/install_plugins.sh [hunk 1]
```diff
@@ -7,20 +7,24 @@ echo "Installing rest service plugins"
...
-plugin_dir=$(python -c "${extract_plugin_script}")
+    plugin_dir=$(python -c "${extract_plugin_script}")
 
-pushd ${plugin_dir}
-sudo $VENV/bin/pip install . {{ plugin.install_args }}
-popd
+    pushd ${plugin_dir}
+        if [[ {{ plugin.source }} == *.wgn ]]; then
+            # install wagon plugin
+            sudo $VENV/bin/pip install {{ plugin.install_args }}
```
Comment (noakup) [inline components/restservice/scripts/install_plugins.sh]: do we really need sudo to run pip install inside a virtualenv?

File: components/restservice/scripts/install_plugins.sh [hunk 2]
```diff
@@ -7,20 +7,24 @@ echo "Installing rest service plugins"
...
+        else
+            sudo $VENV/bin/pip install . {{ plugin.install_args }}
```
Comment (noakup) [inline components/restservice/scripts/install_plugins.sh]: same

File: components/restservice/scripts/install_plugins.sh (hunk 1 above)
Comment (adamlavie) [inline components/restservice/scripts/install_plugins.sh]: We do, actually. It's required to use sudo to install dependencies in the manager env

File: components/restservice/scripts/install_plugins.sh (hunk 2 above)
Comment (adamlavie) [inline components/restservice/scripts/install_plugins.sh]: same..

**Pull request closed without merging**